### PR TITLE
e2e: add member remove test

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: e2e/ctl_v3_member_test.go [hunk 1]
```diff
@@ -32,3 +41,72 @@ func ctlV3MemberList(cx ctlCtx) error {
...
+
+func getMemberList(cx ctlCtx) (memberList, error) {
+	cmdArgs := append(cx.PrefixArgs(), "--write-out", "json", "member", "list")
+
+	proc, err := spawnCmd(cmdArgs)
+	if err != nil {
+		return memberList{}, err
+	}
+	var txt string
+	txt, err = proc.ExpectFunc(func(txt string) bool { return true })
```
Comment (heyitsanthony) [inline e2e/ctl_v3_member_test.go]: `Expect("members")`?

File: e2e/ctl_v3_member_test.go [hunk 2]
```diff
@@ -32,3 +41,72 @@ func ctlV3MemberList(cx ctlCtx) error {
 	}
 	return spawnWithExpects(cmdArgs, lines...)
 }
+
+type memberList struct {
```
Comment (heyitsanthony) [inline e2e/ctl_v3_member_test.go]: why a special struct? MemberListResponse can already be serialized/deserialized in json

Comment (gyuho) [inline e2e/ctl_v3_member_test.go]: Sure, will just import that `struct`. thanks

**Comment by gyuho** [comment]
@heyitsanthony All fixed up. PTAL. Thanks.

File: e2e/ctl_v3_member_test.go [hunk 3]
```diff
@@ -32,3 +41,67 @@ func ctlV3MemberList(cx ctlCtx) error {
...
+	if err != nil {
+		return etcdserverpb.MemberListResponse{}, err
+	}
+	if err = proc.Close(); err != nil {
+		return etcdserverpb.MemberListResponse{}, err
+	}
+
+	resp := etcdserverpb.MemberListResponse{}
+	dec := json.NewDecoder(strings.NewReader(txt))
+	for {
```
Comment (heyitsanthony) [inline e2e/ctl_v3_member_test.go]: is looping necessary?

**Comment by heyitsanthony** [comment]
lgtm aside from the `for` thing

**Comment by gyuho** [comment]
Just removed `for` loop as you suggested. Merging in greens. Thanks!

**Pull request merged**